newIDE/app/src/InstructionOrExpression/EnumerateExpressions.spec.js

```diff
@@ -48,13 +48,16 @@ describe('EnumerateExpressions', () => {
     );
 
     expect(
-      filterExpressions(platformerObjectBehaviorExpressions, 'JumpSpeed')
+      filterExpressions(platformerObjectBehaviorExpressions, 'CurrentJumpSpeed')
     ).toHaveLength(1);
     expect(
       filterExpressions(platformerObjectBehaviorExpressions, 'JumpSpeed')
     ).toContainEqual(
       expect.objectContaining({
         type: 'JumpSpeed',
+      }),
+      expect.objectContaining({
+        type: 'CurrentJumpSpeed',
       })
     );
```

Comment:
Unfortunately you can't pass two arguments to toContainEqual.

Instead, save the result of filterExpressions in a variable, then do the checks:
`...toHaveLength(2)`
`...toContainEqual( expect.objectContaining({...)`
`...toContainEqual( expect.objectContaining({...)`